src/Libraries/Microsoft.Extensions.AI.Abstractions/Image/ImageOptions.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Drawing;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>Represents the options for an image generation request.</summary>
+[Experimental("MEAI001")]
+public class ImageOptions
```

Comment:
Is this too general?  I know this is what folks agreed to in our offline discussion - let me know if we instead want to add `Generation` in there.  Same for request/response/update.